### ISSUE TITLE
Crashes on node v12

### ISSUE THREAD

**Issue by jaydp17** [dougmoscrop/serverless-plugin-split-stacks#62]
In the below code we check for `error.message` to end with the `customPath`.

https://github.com/dougmoscrop/serverless-plugin-split-stacks/blob/53326465b7955f9d0e3ebeef1f0d310f5197cab5/lib/migration-strategy/custom.js#L16

But starting with node.js v12, for `MODULE_NOT_FOUND` errors the error message also includes the require stack (https://github.com/nodejs/node/pull/25690), which breaks the expected behavior that we want. 

```js
// e.message before node v12
"Cannot find module '/<serviceDirectory>/stacks-map.js'"
```

```js
// e.message in node v12
"Cannot find module '/<serviceDirectory>/stacks-map.js'
Require stack:
- /<serviceDirectory>/node_modules/serverless-plugin-split-stacks/lib/migration-strategy/custom.js
- /<serviceDirectory>/node_modules/serverless-plugin-split-stacks/lib/migrate-new-resources.js
- /<serviceDirectory>/node_modules/serverless-plugin-split-stacks/split-stacks.js
- /<serviceDirectory>/node_modules/serverless/lib/classes/PluginManager.js
- /<serviceDirectory>/node_modules/serverless/lib/Serverless.js
- /<serviceDirectory>/node_modules/serverless/lib/utils/autocomplete.js
- /<serviceDirectory>/node_modules/serverless/bin/serverless"
```

**Comment by johndww** [comment]
@dougmoscrop Is it possible to cut a release for this? I can't figure out how to include the fix in my deployment in package.json.

**Comment by dougmoscrop** [comment]
Whoops! I will publish a release now.

For future reference if you're in a pinch, you can use a github reference as a version in package.json:

```json
"serverless-plugin-split-stacks": "github:dougmoscrop/serverless-plugin-split-stacks#tag-or-sha"
```

**Comment by johndww** [comment]
@dougmoscrop thanks very much! Good to know for the future.

**Comment by dougmoscrop** [comment]
Published 1.7.3 with the fix
